### DISCUSSION TITLE
How many RAMs (Not VRAMs) required for this?

### DISCUSSION THREAD

**Discussion by MeemeeLab** [hlky/stable-diffusion#278] [Q&A] [answered]
I have RTX 3060 that has 12GB VRAMs, but I only have 8 GB RAMs so I'm thinking to upgrade my RAM.
How much RAMs I should add on my pc?

**Comment by SafentisFox** [comment, marked as answer]
Have you tried running it as is? 
Personally with my 16bg of ram and 6gb of vram, it generally uses 4-5gb of ram.

**Reply by MeemeeLab** [reply]
Somehow my memory broke and I only have 4GB, so I can't test it right now...
I guess I have to upgrade to 16GB

**Reply by Oros42** [reply]
@MeemeeLab you can increase your swapfile (https://learnubuntu.com/add-swap-ubuntu/).
Yes it's very slow but it can work.
